linodecli/configuration/helpers.py

```diff
@@ -45,6 +61,30 @@ def _get_config(load=True):
 
     return conf
 
+def _check_browsers():
+    # let's see if we _can_ use web
+    result = True
+    try:
+        webbrowser.get()
+    except webbrowser.Error:
+        # there are no browsers installed
+        result = False
```

Comment:
I think we can return false here and drop `result` from the function altogether.
```suggestion
        return False
```